### ISSUE TITLE
Popup: Floating elements not positioned properly

### ISSUE THREAD

**Issue by qooban** [Semantic-Org/Semantic-UI-React#4092]
## Bug Report

In new `Popup` component floated elements are not positioned properly. That is probably because of two `div` blocks instead of one `div` block (in v1.x) rendered as part of `Popup` component.

### Steps
Just try:
```
<Popup trigger={<Button>Hover me</Button>}>
    <Button floated="right">Test</Button>
</Popup>
```
Codesanbox with the same example available below.

### Expected Result
Button "Test" is inside the popup.

### Actual Result
Button "Test" is partially outside of the popup.

### Version
2.0.0

### Testcase
https://codesandbox.io/s/semantic-ui-react-forked-he9zh

**Comment by layershifter** [comment]
@qooban Thanks for reporting, I fixed the issue in #4094 (visual tests were also added). I expect that we will release it in `2.0.1` on Monday.

**Comment by layershifter** [comment]
Released in `semantic-ui-react@2.0.1` 🚢
